apps/cowswap-frontend/src/modules/permit/utils/handlePermit.ts

```diff
@@ -41,6 +42,6 @@ export async function handlePermit(params: HandlePermitParams): Promise<AppDataI
     return replaceHooksOnAppData(appData, hooks, filterPermitSignerPermit)
   } else {
     // Otherwise, pass along exiting hooks, minus permit
-    return replaceHooksOnAppData(appData, removePermitHookFromHooks(typedHooks))
+    return removePermitHookFromAppData(appData, typedHooks)
```

Comment:
As discussed with Bleu, there might be other of permits included. In that case we'll need to pass a parameter down as to which target shouldn't be allowed to be included (Settlement contract).
Not necessarily in this PR.